public/assets/javascript/app.js

```diff
@@ -1,18 +1,18 @@
 /* globals $, L, moment, GTFS_BASE_URL */
 
-var refreshRate = 5 * 1000
-var refreshAttempts = 1
+const refreshRate = 5 * 1000
+let refreshAttempts = 1
 
-var markers = {}
-var locationMarker = {}
-var tripUpdates = {}
-var routesData = {}
-var agenciesData = {}
-var routeShapes = {}
-var stopsData = {}
+const markers = {}
+let locationMarker = {}
```

Comment:
This could be converted to a `const` if L64-67 are instead broken out into two assignments rather than reassigning the whole variable.